services/iiif/image.js

```diff
@@ -0,0 +1,40 @@
+const axios = require("axios");
```

Comment:
General comment about image.js in general. This is fantastic for Canopy. I really want to look into integrating it into Nectar in some way. I also think it could be beneficial for work on https://github.com/IIIF-Commons/thumbnail-panel